stories/annotations/mixed.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { Meta } from '@storybook/react';
+
+import {
+  Line,
+  AnnotationLineProps,
+} from '../../src/components/Annotations/Line';
+
+import { AnnotationPlot } from './annotation.data';
+
+export default {
+  title: 'API/Annotations',
+  component: Line,
+  args: {
+    x1: '40%',
```

Comment:
Can you maybe add another annotation that uses 0% and 100% so that's it's easier to see that it works properly?

Reply:
added